### PR TITLE
feat(headless): include the loadgenericanalyticsactions action loader in the case assist & insight headless bundle

### PR DESCRIPTION
**Pull request by SimonMilord**
[SFINT-4918](https://coveord.atlassian.net/browse/SFINT-4918)

I want to be able to send my own custom analytics events when using Quantic to implement my case assist flow.

- loadGenericAnalyticsActions function is currently included only in the search bundle, we need to include it in the Headless bundle of other use cases.
- we need to expose loadGenericAnalyticsActions for the insight use case as well

As you can see below, we can have access to loadGenericAnalyticsActions

CASE ASSIST: 

Screenshot 2023-04-17 at 5 29 35 PM

INSIGHT: 

Screenshot 2023-04-17 at 5 29 10 PM
ericAnalyticsActions in quantic in case assist & insight components

[SFINT-4918]: https://coveord.atlassian.net/browse/SFINT-4918?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/headless/doc-parser/use-cases/insight.ts [hunk 1]
```diff
@@ -108,6 +108,9 @@ const actionLoaders: ActionLoaderConfiguration[] = [
   {
     initializer: 'loadCaseContextActions',
   },
+  {
+    initializer: 'loadGenericAnalyticsActions',
+  },
```
Comment (btaillon) [inline packages/headless/doc-parser/use-cases/insight.ts:113]: I just noticed that when I added more action loaders to `insight.index.ts` last time, I forgot to add them here. Could add them in this PR?

**Review by btaillon — APPROVED** [review]

File: packages/headless/doc-parser/use-cases/insight.ts (hunk 1 above)
Comment (SimonMilord) [reply packages/headless/doc-parser/use-cases/insight.ts:131]: yes just added them! :)

File: packages/headless/doc-parser/use-cases/insight.ts [hunk 2]
```diff
@@ -96,18 +96,39 @@ const actionLoaders: ActionLoaderConfiguration[] = [
...
     initializer: 'loadInsightSearchActions',
   },
+  {
+    initializer: 'loadQuerySetActions',
+  },
   {
     initializer: 'loadInsightAnalyticsActions',
   },
   {
-    initializer: 'loadInsightSearchAnalyticsActions',
```
Comment (SimonMilord) [inline packages/headless/doc-parser/use-cases/insight.ts:103]: changed them to be in the same order as they are exported

Comment (SimonMilord) [inline packages/headless/doc-parser/use-cases/insight.ts:103]: changed them to be in the same order as they are exported in insight.index.ts for readability

**Review by nathanlb — APPROVED** [review]

**Pull request merged**